bitnami/common/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: common
 # Please make sure that version and appVersion are always the same.
-version: 0.6.2
-appVersion: 0.6.2
+version: 0.6.3
+appVersion: 0.6.3
```

Comment:
Why are you changing the app version?

Reply:
It is because pipeline requirements for this special chart